customize-layout-and-css.patch

Customize the layout and style sheet

diff --git a/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html b/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html
index f35a892..9f0b499 100644
--- a/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html
+++ b/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html
@@ -114,7 +114,7 @@
           {% block breadcrumbs %}
           <div role="navigation" aria-label="breadcrumbs navigation">
             <ol class="breadcrumb">
-              <li><a href="{{ pathto(master_doc) }}">Docs</a></li>
+              <li><a href="{{ pathto(master_doc) }}">Home</a></li>
               {% for doc in parents %}
                 <li><a href="{{ doc.link|e }}">{{ doc.title }}</a></li>
               {% endfor %}
diff --git a/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html b/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html
index e7c949c..00dec78 100644
--- a/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html
+++ b/guzzle_sphinx_theme/guzzle_sphinx_theme/logo-text.html
@@ -3,6 +3,9 @@
   {# Not strictly valid HTML, but it's the only way to display/scale it properly, without weird scripting or heaps of work #}
   <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" />
   </a>
+  <center><a href="https://github.com/opencomputeproject/onie" class="logo">
+  <svg class="octicon-mark-github" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"/></svg>
+  opencomputeproject/onie</a></center>
 {% else %}
   <a href="{{ homepage() }}" class="text-logo">{{ theme_project_nav_name or shorttitle }}</a>
 {% endif %}
diff --git a/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t b/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t
index 5379195..94cd58e 100644
--- a/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t
+++ b/guzzle_sphinx_theme/guzzle_sphinx_theme/static/guzzle.css_t
@@ -50,10 +50,13 @@ h1 > a, h2 > a, h3 > a, h4 > a, h5 > a, h6 > a {
 }
 
 h1, h2, h3, h4, h5, h6 {
-  color: black;
-  font-weight: normal;
+  color: #111;
   padding: 0;
-  font-family: "Source Serif Pro", "serif";
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+  font-weight: 300;
+  -webkit-text-size-adjust: 100%;
 }
 
 h1, h2, h3 {
@@ -406,6 +409,12 @@ table.two-column {
   height: 40px;
 }
 
+/* github svg icon styling
+-------------------------------------------------- */
+svg.octicon-mark-github path {
+  fill: #888;
+}
+
 /* Footer styling
 -------------------------------------------------- */
 
@@ -418,10 +427,6 @@ div.footer {
   width: 100%;
 }
 
-div.footer a {
-  color: #888;
-}
-
 /* -- relbar ---------------------------------------------------------------- */
 
 div.related {
